src/lib/config.js

```diff
@@ -101,16 +137,37 @@ function validateLocalEnvConfig () {
   }
 }
 
+function validateNotificationEnv () {
+  // Validate notification signing public keys
+  const notifications = parseNotificationSignEnv()
+
+  if (notifications.must_verify) {
+    const ledgers = parseLedgers()
+    for (let ledgerObj of ledgers) {
+      const uri = ledgerObj.ledger
+      if (notifications.keys[uri] === undefined) {
+        throw new Error(`Missing notification signing keys for ledger: ${uri}`)
```

Comment:
The connector will fail to start, so it's not necessary to create a typed error here, just a good error message.